Patches/Advanced Mechanoid Warfare/Patches/Patch_Ranged.xml

```diff
@@ -0,0 +1,886 @@
+						<hasStandardCommand>true</hasStandardCommand>
+						<range>32</range>
+						<burstShotCount>4</burstShotCount>
+						<warmupTime>2.3</warmupTime>
+						<noiseRadius>7</noiseRadius>
+						<ai_IsBuildingDestroyer>true</ai_IsBuildingDestroyer>
+						<soundCast>DiskLaunch</soundCast>
+						<targetParams>
+							<canTargetLocations>true</canTargetLocations>
+						</targetParams>
+						<defaultProjectile>BreacherGrenade</defaultProjectile>
+						<ignorePartialLoSBlocker>true</ignorePartialLoSBlocker>
+					</Properties>
+					<FireModes>
+						<aiAimMode>AimedShot</aiAimMode>
+					</FireModes>
+					<weaponTags>
+						<li>CE_AI_Launcher</li>
+					</weaponTags>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[defName="Bullet_PlasmaSpit"]/thingClass</xpath>
+				<value>
+					<thingClass>CombatExtended.ProjectileCE_Explosive</thingClass>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[defName="Bullet_PlasmaSpit"]/projectile</xpath>
+				<value>
+					<projectile Class="CombatExtended.ProjectilePropertiesCE">
+					  <damageDef>Bomb</damageDef>
+					  <damageAmountBase>309</damageAmountBase>
+					  <explosionRadius>3.2</explosionRadius>
+					  <speed>75</speed>
+					  <applyDamageToExplosionCellsNeighbors>true</applyDamageToExplosionCellsNeighbors>
+					  <dropsCasings>false</dropsCasings>
+					</projectile>
+				</value>
+			</li>
+			
+
+			<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+				<defName>Gun_PlasmaSpitInternal</defName>
+					<statBases>
+						<SightsEfficiency>1.0</SightsEfficiency>
+						<ShotSpread>0.14</ShotSpread>
+						<SwayFactor>1.01</SwayFactor>
+						<Bulk>3.5</Bulk>
+						<Mass>6</Mass>
+						<RangedWeapon_Cooldown>4</RangedWeapon_Cooldown>
+					</statBases>
+					<Properties>
+						<recoilAmount>1.53</recoilAmount>
+						<verbClass>CombatExtended.Verb_ShootCE</verbClass>
+						<hasStandardCommand>true</hasStandardCommand>
+						<defaultProjectile>Bullet_PlasmaSpit</defaultProjectile>
+						<warmupTime>4.3</warmupTime>
+						<range>54</range>
+						<!--<ai_AvoidFriendlyFireRadius>5</ai_AvoidFriendlyFireRadius>-->
+						<soundCast>HeavyLaunch</soundCast>
+						<soundCastTail>GunTail_Heavy</soundCastTail>
+						<muzzleFlashScale>7</muzzleFlashScale>
+						<ignorePartialLoSBlocker>true</ignorePartialLoSBlocker>
+					</Properties>
+					<FireModes>
+						<aiAimMode>AimedShot</aiAimMode>
+					</FireModes>
+					<weaponTags>
+						<li>CE_AI_Launcher</li>
+					</weaponTags>
+			</li>
+			
+			
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[defName="Bullet_KineticWing"]/thingClass</xpath>
+				<value>
+					<thingClass>CombatExtended.ProjectileCE_Explosive</thingClass>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/ThingDef[defName="Bullet_KineticWing"]/projectile</xpath>
+				<value>
```

Comment:
Is this meant to have a visible projectile sprite? It's currently invisible.

Reply:
It's sprite is a black line. I tried making it bigger with drawSize but it didn't work.